### ISSUE TITLE
Available unlabeled data

### ISSUE THREAD

**Issue by cuishuhao** [VisionLearningGroup/visda21-dev#3]
Hello, can we use the unlabeled data in validation set in DA process?

**Comment by samarth4149** [comment]
Hi, you can do so while you develop your model and chose hyperparameters, but not during evaluation on the final test split (in which case unlabeled data from the test split can be used).
